website/src/libs/api-v3/setupRoutes.js

```diff
@@ -14,7 +14,7 @@ fs
     let controller = require(CONTROLLERS_PATH + fileName); // eslint-disable-line global-require
 
     _.each(controller, (action) => {
-      let {method, url, middlewares, handler} = action;
+      let {method, url, middlewares = [], handler} = action;
```

Comment:
yep, I didn't add a default for url, method and handler because those are not optional